src/test/java/cruise/umple/sample/downloader/ConsoleMainTest.java

```diff
@@ -17,19 +14,16 @@
 import com.google.inject.Inject;
 import com.google.inject.Provider;
 
-import cruise.umple.sample.downloader.ConsoleMain.ImportRuntimeData;
+import cruise.umple.sample.downloader.consistent.ImportRepository;
+import cruise.umple.sample.downloader.consistent.ImportRepositorySet;
 import cruise.umple.sample.downloader.util.MockDocumentFactoryModule;
-import cruise.umple.sample.downloader.util.Pair;
 
 /**
  * Created by kevin on 15-02-23.
  */
-@Guice(modules={DownloaderModule.class, MockDocumentFactoryModule.class})
+@Guice(modules={MockDocumentFactoryModule.class})
```

Comment:
I don't think it does (or at least it's undocumented):
![screen shot 2015-03-17 at 16 57 51](https://cloud.githubusercontent.com/assets/302365/6697428/da695cea-ccc6-11e4-88e0-88b26a74d3cd.png)
I looked through the grammar, and there is no notion either. 

As for modelling, that was my thought too. If we support it at the "meta" level and let implementations lower it down as available, I think it solves the niche issue. 

Another note though, I can't seem to find any way to actually access or modify a ctor for an object in Umple aside from defining a new constructor which loses language generality a bit. This is a small "wart" though, so probably negligible.